packages/util/src/kzg.ts

```diff
@@ -32,10 +37,13 @@ export let kzg: Kzg = {
 }
 
 /**
+ * @deprecated This initialization method is deprecated since trusted setup loading is done directly in the reference KZG library
+ * initialization or should othewise be assured independently before KZG libary usage.
+ *
  * @param kzgLib a KZG implementation (defaults to c-kzg)
- * @param trustedSetupPath the full path (e.g. "/home/linux/devnet4.txt") to a kzg trusted setup text file
+ * @param a dictionary of trusted setup options
  */
-export function initKZG(kzgLib: Kzg, trustedSetupPath?: string) {
+export function loadKZG(kzgLib: Kzg, _trustedSetupPath?: string) {
   kzg = kzgLib
```

Comment:
```suggestion
```
i don't think we need this, as well as the exported kzg object because it was earlier only used in helpers which you have updated